### PR TITLE
Mercat update

### PR DESCRIPTION
**Pull request by ajaypanyala**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 os:
+  - linux
```
Comment (bgruening) [inline .travis.yml]: Can you please revert this?

File: recipes/mercat/meta.yaml [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-#  git_rev: 0.1
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/pnnl/mercat/archive/0.1.tar.gz
+  md5: bce67a7bcb58c8daed4a00b377dad097
 
-# build:
-#  number: 0.1
 
 build:
+  number: 0
```
Comment (bgruening) [inline recipes/mercat/meta.yaml]: You need to increase the version number if you want to have a new package.

File: .travis.yml (hunk 1 above)
Comment (ajaypanyala) [inline .travis.yml]: This line was present in the original travis.yml file in the bioconda repo. I removed it in an earlier commit and reverted it back later to the original one.

File: recipes/mercat/meta.yaml [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
-#  git_rev: 0.1
+  fn: {{ name|lower }}_{{ version }}.tar.gz
+  url: https://github.com/pnnl/mercat/archive/0.2.tar.gz
+  md5: bce67a7bcb58c8daed4a00b377dad097
 
-# build:
-#  number: 0.1
 
 build:
+  number: 1
```
Comment (druvus) [inline recipes/mercat/meta.yaml]: Build number should start with 0 for new versions.

**Pull request merged**